queries.sql

```diff
@@ -14,3 +14,38 @@ SELECT * FROM animals WHERE neutered=TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT DELETE_AFTER_JAN_22;
+UPDATE animals SET weight_kg = (weight_kg * -1);
+ROLLBACK TO DELETE_AFTER_JAN_22;
+UPDATE animals SET weight_kg = (weight_kg * -1) WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals;
+SELECT name, MAX(escape_attempts) FROM animals GROUP BY name ORDER BY max DESC;
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01'GROUP BY species;
```

Comment:
- [ ] As specified in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), " `What is the average number of escape attempts per animal type of those born between 1990 and 2000?` ", kindly update your range to include the end of 2000. Some like so: 
```
   WHERE  date_of_birth >= '1990/01/01' AND date_of_birth <= '2000/31/12';
```